migrations/175_workspace_logs__create.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TYPE enum_workspace_log_log_level AS ENUM ('debug', 'error', 'info', 'warn');
```

Comment:
It's important that these should be ordered by decreasing level, so we can easily find all things with `level < warn`, for example.